src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -0,0 +1,17 @@
+package wolox.training;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@EnableJpaRepositories("java.wolox.training.repositories")
```

Comment:
Remember change the mapping package to wolox.training.repossitories

Reply:
It was solve on:
## Summary
 - Fix  pull request comment about JPARepository 
 -## Trello Card 
-https://trello.com/c/ekXiJvjm/15-crear-un-repositorio-de-books